io-engine/src/bdev/nexus/nexus_channel.rs

```diff
@@ -38,6 +41,15 @@ impl<'n> Debug for NexusChannel<'n> {
     }
 }
 
+/// Channel I/O disposition.
+#[derive(Debug, Copy, Clone)]
+pub enum IOMode {
```

Comment:
```suggestion
pub enum IoMode {
```

we use "Io*" elsewhere, eg: IoType/IoChannel/IoType